test/box-luatest/downgrade_test.lua

```diff
@@ -1030,7 +1030,9 @@ g.test_downgrade_func_trigger = function(cg)
         check_before_downgrade(true)
 
         -- Drop the functions with trigger and check if downgrade collects
-        -- no issues and works correctly
+        -- no issues and works correctly. Upgrade is needed to be able to
+        -- drop the functions.
+        box.schema.upgrade()
```

Comment:
Please explain in more detail this change.

Reply:
Done.